src/main/groovy/jenkins/automation/utils/GhUtils.groovy

```diff
@@ -0,0 +1,62 @@
+package jenkins.automation.utils
+
+/**
+ * Utility class for internally hosted GitHub Enterprise interaction
+ */
+
+class GhUtils {
+
+/**
+ *
+ * Utility method to watch a GitHub Enterprise or GitHub.com project and trigger on PR creation
+ * This is a wrapper for the <a href="https://wiki.jenkins.io/display/JENKINS/GitHub+pull+request+builder+pluginGitHub">Pull Request Builder Plugin</a>.
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#using-gh-pr-watcher" target="_blank">example</a>
+ * @param  context delegate
+ * @param  ghProject A reference to the GHE project as 'username/reponame'
+ * @param  ghHostname The GitHub hostname, i.e. github.org.tld, or github.com
+ * @param  ghAuthId the Jenkins credential ID for GitHub authentication
+ *         see <a href="https://github.com/jenkinsci/ghprb-plugin/blob/master/README.md#credentials">the plugin docs</a>
+ * @param  ghPermitAll  optional Whether or not to allow all contributors to build a PR
+ * @param  ghPrHooks  optional Whether or not to install webhooks in the remote repo
+ * @param  ghPrCron  optional For use when ghPrHooks is false. Uses standard cron syntax
+ * @param  ghPrOrgList  optional A string of organizations to whitelist for PRs
+ */
+    static void ghPrWatcher(context,
+                            String ghProject,
+                            String ghHostname,
+                            String ghAuthId,
+                            Boolean ghPermitAll = true,
```

Comment:
I think this should default to false. Especially on gh.com, permitting all is not something we should do most likely. We should force a deliberate opt-on on this

Reply:
I'll lean on your experience here, and I see the wisdom.